libraries/AP_Param/AP_Param.cpp

```diff
@@ -2395,6 +2406,9 @@ void AP_Param::load_embedded_param_defaults(bool last_pass)
             }
             continue;
         }
+        if (idx >= param_overrides_len) {
+            INTERNAL_ERROR(AP_InternalError::error_t::flow_of_control);
```

Comment:
Same here - do we want to just rumble on in this case?

Reply:
removed - see comment below